smart-contracts/contracts/cl-vault/src/test_tube/proptest.rs

```diff
@@ -466,7 +473,7 @@ mod tests {
     // get_percentage generates a list of random percentages used to calculate deposit_amount,
     // withdraw_amount, and newers lower and upper ticks based on the previous values
     prop_compose! {
-        fn get_percentage_list()(list in prop::collection::vec(1.0..100.0, ITERATIONS_NUMBER..ITERATIONS_NUMBER+1)) -> Vec<f64> {
+        fn get_percentage_list()(list in prop::collection::vec(1.0..10.0, ITERATIONS_NUMBER..ITERATIONS_NUMBER+1)) -> Vec<f64> {
```

Comment:
@magiodev  why was this set to 100? I believe my tests were failing due to insufficient funds because of this.

Reply:
this percentage list array is used by every action. the test design is to first take the ITERATIONS_NUMBER, with this value creating a list of random percentages between 1 and 100 and use it for the random action we pick during an iteration.
effectively it could lead to errors as if during an iteration we deposit the 100 percent of a user funds and we try it again later, it should be capped to 99, but it worked till now.